src/index.html

```diff
@@ -3,10 +3,381 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="./main.scss">
   </head>
+
+  <header class="header">
+    <a target="_blank" href="#home">
+      <img
+        class="logo"
+        src="images/logo.png"
+        alt="Moyo logo"
+      >
+    </a>
+    <nav class="nav">
+      <ul class="nav__list">
+        <li class="nav__item">
+          <a href="#apple"class="nav__link is-active">apple</a>
+        </li>
+        <li class="nav__item">
+          <a href="#samsung"class="nav__link">samsung</a>
+        </li>
+        <li class="nav__item">
+          <a href="#smartphones"class="nav__link">smartphones</a>
+        </li>
+        <li class="nav__item">
+          <a
+            href="#laptops&computers"
+            class="nav__link"
+            data-qa="nav-hover"
+          >
+          laptops & computers
+          </a>
```

Comment:
Check indents for the content
```suggestion
          >
            laptops & computers
          </a>
```